app/services/card_comparator/result.rb

```diff
@@ -0,0 +1,14 @@
+class CardComparator::Result < Struct.new(:diff)
```

Comment:
Use nested module/class definitions instead of compact style.<br>Don't extend an instance initialized by `Struct.new`.